{
	"deprecated": false,
	"cpeName": "cpe:2.3:h:snom:snom_360:-:*:*:*:*:*:*:*",
	"cpeNameId": "47957453-D0F5-4FB6-9724-5E0FD18EFFD5",
	"created": "2024-02-07T21:02:50.350",
	"lastModified": "2024-02-07T21:02:50.350",
	"titles": [
		{
			"title": "Snom 360 ",
			"lang": "en"
		}
	],
	"refs": [
		{
			"ref": "https://exchange.xforce.ibmcloud.com/vulnerabilities/52424",
			"type": "Advisory"
		},
		{
			"ref": "https://www.snom.com/en/",
			"type": "Vendor"
		}
	]
}
